docs/index.md

```diff
@@ -11,10 +11,13 @@
 
 ![Ui](images/Ui.png)
 
-**AddressBook is a desktop application for managing your contact details.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
+**HomeBoss is a desktop application for home businesses who need to manage their deliveries for their customers.** 
+While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
 
-* If you are interested in using AddressBook, head over to the [_Quick Start_ section of the **User Guide**](UserGuide.html#quick-start).
-* If you are interested about developing AddressBook, the [**Developer Guide**](DeveloperGuide.html) is a good place to start.
+* If you are interested in using HomeBoss,
+  head over to the [_Quick Start_ section of the **User Guide**](UserGuide.html#quick-start).
```

Comment:
Should this be the getting started instead

Reply:
Good catch! Have changed it accordingly.